doc/cephadm/adoption.rst

```diff
@@ -75,11 +75,11 @@ Adoption process
 #. Generate an SSH key::
 
      # ceph cephadm generate-key
-     # ceph cephadm get-pub-key > ceph.pub
+     # ceph cephadm get-pub-key > ~/cephadm_ssh_key.pub
```

Comment:
Hm we already have `ceph.pub` hardcoded: https://github.com/ceph/ceph/search?q=%22ceph.pub%22

Hm. I have the fear that we might irrigate users if we suddenly have different names for the same thing.